docs/source/usage/cli.rst

```diff
@@ -216,6 +216,11 @@ In order to access a node using a serial console, the admin or owner must config
 Instructions for this can be found under `Configuring Web or Serial Console`_ in Ironic's documentation.
 Once the node is properly configured, the console can be enabled and disabled as needed.
 
+Following is the pre-requisite for functioning of serial console:
+
+* The ``ipmi_terminal_port`` port must be unique per node/controller.
```

Comment:
It should be noted that only the admin or node owner can set this value. Also, let's just say that the port must be unique, period.